pkg/network/ebpf/c/protocols/https.h

```diff
@@ -152,15 +160,32 @@ static __always_inline void map_ssl_ctx_to_sock(struct sock *skp) {
  */
 static __always_inline tls_offsets_data_t* get_offsets_data() {
     struct task_struct *t = (struct task_struct *) bpf_get_current_task();
+    go_tls_offsets_data_key_t key;
+    dev_t dev_id;
+
+#if COMPILE_CORE
```

Comment:
Do you need the code in the `#else` clause? `BPF_CORE_READ_INTO` would work on runtime compiled as well.

Reply:
good catch! I'll update it